srcpkgs/mstflint/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'mstflint'
+pkgname=mstflint
+version=4.13.3
+revision=1
+subvers="-2"
+build_style=gnu-configure
+configure_args="--enable-cs --enable-fw-mgr --enable-xml2"
+makedepends="boost1.69-devel libcurl-devel libibmad-devel liblzma-devel libxml2-devel zlib-devel"
```

Comment:
s/boost1.69/boost/

Reply:
If I change it to boost, I get this error:

`checking for regcompA in -lboost_regex... no
configure: error: cannot find boost_regex regcompA() function. Try re-installing the library...
=> ERROR: mstflint-4.13.3_1: do_configure: '${configure_script} ${configure_args}' exited with 1
=> ERROR:   in do_configure() at common/build-style/gnu-configure.sh:7`

Reply:
Nevermind, I am an idiot.  Figured it out.